xml/ssh_banners.xml

```diff
@@ -439,6 +471,18 @@ fingerprint SSH servers.
     <param pos="0" name="os.family" value="RouterOS"/>
     <param pos="0" name="os.product" value="RouterOS"/>
   </fingerprint>
+  <fingerprint pattern="^OpenSSH_(.*)-HipServ$">
+    <description>OpenSSH on HipServ</description>
+    <example service.version="4.3">OpenSSH_4.3-HipServ</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="OpenBSD"/>
+    <param pos="0" name="service.family" value="OpenSSH"/>
+    <param pos="0" name="service.product" value="OpenSSH"/>
+    <param pos="0" name="os.vendor" value="HipServ"/>
+    <param pos="0" name="os.device" value="Router"/>
+    <param pos="0" name="os.family" value="RouterOS"/>
+    <param pos="0" name="os.product" value="RouterOS"/>
```

Comment:
Yep, it is.  In the description I put that I wasn't sure about HipServe because it's a NAS but I couldn't find a suitable device for that so I called it a Router for now.  What would you recommend for that?